syfoalerts.yaml

```diff
@@ -54,29 +61,50 @@ spec:
       severity: danger
     - alert: syfovarsel-lytter-stoppet
       expr: sum(syfovarsel_kafkalytter_stoppet_total) > 0
-      for: 3m
+      for: 5m
       description: "{{ $labels.app }} sin kafkalytter har stoppet i {{ $labels.kubernetes_namespace }}"
       action: "Kafkalytter stoppet på syfovarsel. Sjekk logger, og evt. `kubectl delete pod {{ $labels.kubernetes_pod_name }}` for å restarte"
       sla: respond within 1h, during office hours
       severity: danger
     - alert: syfoaltinn-lytter-stoppet
       expr: sum(syfoaltinn_kafkalytter_stoppet_total) > 0
-      for: 3m
+      for: 5m
       description: "{{ $labels.app }} sin kafkalytter har stoppet i {{ $labels.kubernetes_namespace }}"
       action: "Kafkalytter stoppet på syfoaltinn. Sjekk logger, og evt. `kubectl delete pod {{ $labels.kubernetes_pod_name }}` for å restarte"
       sla: respond within 1h, during office hours
       severity: danger
     - alert: syfosyketilfelle-lytter-stoppet
       expr: sum(syfosyketilfelle_kafkalytter_stoppet_total) > 0
-      for: 3m
+      for: 5m
       description: "{{ $labels.app }} sin kafkalytter har stoppet i {{ $labels.kubernetes_namespace }}"
       action: "Kafkalytter stoppet på syfosyketilfelle. Sjekk logger, og evt. `kubectl delete pod {{ $labels.kubernetes_pod_name }}` for å restarte"
       sla: respond within 1h, during office hours
       severity: danger
     - alert: syfoservicestrangler-lytter-stoppet
       expr: sum(syfoservicestrangler_kafkalytter_stoppet_total) > 0
-      for: 3m
+      for: 5m
       description: "{{ $labels.app }} sin kafkalytter har stoppet i {{ $labels.kubernetes_namespace }}"
       action: "Kafkalytter stoppet på syfoservicestrangler. Sjekk logger, og evt. `kubectl delete pod {{ $labels.kubernetes_pod_name }}` for å restarte"
       sla: respond within 1h, during office hours
       severity: danger
+    - alert: syfosmvarsel-avvist-sm-usynk
+      expr: (sum(syfosmvarsel_avvist_sykmelding_mottatt_count) - sum(syfosmvarsel_avvist_sykmelding_opprettetvarsel_count)) > 100
```

Comment:
Spent hvordan denne oppfører seg med de unøyaktighetene vi ser på talla. Kan vi ende opp med en negativ sum om de er litt unøyaktige?

Reply:
Kanskje du kan ta en form for `abs()` så den tåler unøyaktighet begge veier?

Reply:
Kanskje, men da er det noe veldig galt med prometheus i så fall.. Vi bør nok ha tallene på et board i tillegg som vi kan se på.